<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>494</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ежедневник</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTableWidget" name="articleBrowser">
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата и время</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Важность</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="add">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCurrent">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tenArticles">
            <property name="text">
             <string>Кнопка которая да</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="editor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Редактирование</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Название мероприятия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name">
             <property name="maxLength">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Дата и врмя проведения</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateTimeEdit" name="dateTimeEvent">
                 <property name="displayFormat">
                  <string>dd.MM.yyyy HH:mm</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Место проведения</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="location">
                 <property name="maxLength">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Протяженность</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTimeEdit" name="timeEvent">
                 <property name="maximumTime">
                  <time>
                   <hour>23</hour>
                   <minute>59</minute>
                   <second>59</second>
                  </time>
                 </property>
                 <property name="displayFormat">
                  <string>HH:mm</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="importance">
                 <property name="title">
                  <string>Важность</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QRadioButton" name="noImportant">
                    <property name="text">
                     <string>Не важно</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="important">
                    <property name="text">
                     <string>Важно</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="veryImportant">
                    <property name="text">
                     <string>Очень важно</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="hasPeriod">
                 <property name="text">
                  <string>Имеется переодичность</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="period">
                 <property name="title">
                  <string>Периодичность</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QRadioButton" name="everyDay">
                    <property name="text">
                     <string>Каждый день</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="acrossDay">
                    <property name="text">
                     <string>Через день</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="acrossTwoDays">
                    <property name="text">
                     <string>Через два дня</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="acrossTreeDays">
                    <property name="text">
                     <string>Через три дня</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="everyWeak">
                    <property name="text">
                     <string>Каждую неделю</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="everyMouth">
                    <property name="text">
                     <string>Раз в месяц</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="everyQuarter">
                    <property name="text">
                     <string>Каждый квартал</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="everyHalfYear">
                    <property name="text">
                     <string>Раз в пол года</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="everyYear">
                    <property name="text">
                     <string>Раз в год</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Кол-во повторений</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="repeatCount">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>366</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="saveButton">
               <property name="text">
                <string>Сохранить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancel">
               <property name="text">
                <string>Отмена</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Чел, ты</string>
    </property>
    <addaction name="StartServer"/>
    <addaction name="StopServer"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="StartServer">
   <property name="text">
    <string>Старт чела</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="StopServer">
   <property name="text">
    <string>Стоп чел</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
